output/specialty-modslot-metadata.json

```diff
@@ -2,47 +2,47 @@
   {
     "season": 4,
     "tag": "outlaw",
-    "compatibleTags": ["forge", "outlaw"],
-    "thisSlotPlugCategoryHashes": [2149155760, 13646368],
-    "compatiblePlugCategoryHashes": [2149155760, 13646368, 65589297],
+    "compatibleTags": ["outlaw", "forge"],
+    "thisSlotPlugCategoryHashes": [13646368, 2149155760],
+    "compatiblePlugCategoryHashes": [13646368, 2149155760, 65589297],
```

Comment:
65589297 should be in the middle?

Reply:
javascript default sort treats numbers as strings. i don't really care as long as results are repeatable.